packages/core/src/utils/markers.ts

```diff
@@ -0,0 +1,108 @@
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+export interface MarkerStyle {
+  strokeWidth?: string | number
+  stroke?: string
+  fill?: string
+}
+
+export type GetMarkerFunction = (style: MarkerStyle, multi: boolean) => Style
+
+export const imgSize = [23, 32]
+export const imgSizeMulti = [35, 32]
```

Comment:
These seem like awfully specific values - please add a comment on why we got the magic here

Reply:
https://github.com/Dataport/polar/pull/42/commits/3b5b80e5b16a7094a6819ca05fabe1a176adc5c8

I don't remember why they're separate to the rest of the svg strings. Maybe they were intended to be configurable at some point. Changing the height values actually scales the markers. We should keep this in mind, should a need arise.